src/test/java/uk/gov/hmcts/reform/em/hrs/controller/HearingRecordingControllerTest.java

```diff
@@ -129,4 +131,37 @@ void testShouldNotExceedOneSecond() throws Exception {
         assertThat(Duration.between(start, end)).isLessThanOrEqualTo(Duration.ofSeconds(1L));
     }
 
+    @Test
+    void testShouldReturnRequestAccepted() throws Exception {
+        final String path = "/segments";
+        ingestionQueue.clear();
+
+        mockMvc.perform(post(path)
+                            .content(convertObjectToJsonString(HEARING_RECORDING_DTO))
+                            .contentType(APPLICATION_JSON_VALUE))
+            .andExpect(status().isAccepted())
+            .andReturn();
+    }
+
+    @Test
+    void testShouldReturnTooManyRequests() throws Exception {
+        final String path = "/segments";
+        clogJobQueue();
+
+        mockMvc.perform(post(path)
+                            .content(convertObjectToJsonString(HEARING_RECORDING_DTO))
+                            .contentType(APPLICATION_JSON_VALUE))
+            .andExpect(status().isTooManyRequests())
+            .andReturn();
+    }
+
+    private void clogJobQueue() {
+        IntStream.rangeClosed(1, 1000)
```

Comment:
why not use a smaller upper limit for the test, rather than creating a 1000 objects

Reply:
The queue capacity is currently not externalised - so that's not an option at the moment.  Will done on a next task.